javascript/ql/test/query-tests/Security/CWE-022/TaintedPath/express.js

```diff
@@ -4,6 +4,16 @@ var express = require("express"),
 let app = express();
 app.use(fileUpload());
 
-app.get("/some/path", function (req, res) {
+app.get("/some/path/1", function (req, res) {
   req.files.foo.mv(req.query.bar);
 });
+
+app.get("/some/path/2", function (req, res) {
+  const p = path.join('.', req.query.filename);
+  if (!p.startsWith('..')) {
+    req.files.foo.mv(p); // OK (but not on windows)
+    res.status(200);
+  } else {
+    res.status(400);
+  }
+});
```

Comment:
Could you move this test to the Windows-path test folder?  
With the test I suggested we can actually test that `req.files.foo.mv(p)` is only flagged by the windows query.